src/styles/index.scss

```diff
@@ -1,3 +1,67 @@
+@import './variables';
+
 body {
   margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  width: 726px;
+  box-sizing: border-box;
+  padding: 10px;
+
+  &__space {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 1px;
+  }
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: $square-size;
+    width: $square-size;
+    box-sizing: border-box;
+    border: 1px solid black;
```

Comment:
It's a good practice to use variables for all values that are used more than once throughout the code. The border value of 1px solid black might be used elsewhere in your other files. Consider moving it to your `variables.scss` file. This will make your code more maintainable as you will only need to change the value in one place should it need to be updated in the future.